packages/qemuqmp/default.nix

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  python3Packages,
+  fetchPypi,
+  lib,
+}:
+with python3Packages;
+  buildPythonPackage rec {
+    pname = "qemu.qmp";
+    version = "0.0.3";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-y8iPvMEV7pQ9hER9FyxkLaEgIgRRQWwvYhrPM98eEBA=";
+    };
+
+    pyproject = true;
+
+    nativeBuildInputs = with python3Packages; [
```

Comment:
avoid with and just use it explicitly for the one package below?

Reply:
Sure, fixed.